packages/slate-env/__tests__/index.test.js

```diff
@@ -14,6 +14,7 @@ const TEST_ENV = {
   [config.envPasswordVar]: '123456789',
   [config.envThemeIdVar]: '987654321',
   [config.envIgnoreFilesVar]: 'config/settings_data.json',
+  [config.envUserEmail]: 'thomas.kelly@shopify.com',
```

Comment:
Can probably keep it consistent and use `test@email.com` like you're doing above in `packages/slate-analytics/__tests__/index.test.js`.